io.welle.welle-gui.yml

```diff
@@ -0,0 +1,103 @@
+app-id: io.welle.welle-gui
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: welle-io
+finish-args:
+  - '--share=network'
+  - '--share=ipc'
+  - '--filesystem=xdg-documents'
+  - '--socket=pulseaudio'
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  - '--device=dri'
+  - '--device=all'
+
+modules:
+  - name: fftw
+    buildsystem: autotools
+    config-opts:
+      - '--enable-shared'
+      - '--disable-static'
+      - '--enable-threads'
+      - '--enable-float'
+    sources:
+      - type: archive
+        url: 'http://fftw.org/fftw-3.3.10.tar.gz'
+        md5: 8ccbf6a5ea78a16dbc3e1306e234cc5c
+
+  - name: libfaad
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: 'https://salsa.debian.org/multimedia-team/faad2.git'
+        tag: debian/2.11.2-1
+
+  - name: libusb
```

Comment:
please use the shared-modules too

Reply:
I don't find faad2 inside the shared-module repo.

Reply:
it's for libusb https://github.com/flathub/shared-modules/blob/master/libusb/libusb.json

Reply:
I wonder sometime if github show different things to different people:

![image](https://github.com/user-attachments/assets/8bbe2679-1d74-4cc5-919d-a9f6725b09a0)

this looked obvious to me.

Reply:
No github is showing the same to me
![grafik](https://github.com/user-attachments/assets/6a612e7b-2699-42de-ba6e-e53f9cb7786c)

But I saw `faad2` first because it is the first highlighted line. But I already changed to the `libusb` shared module.